src/DetailView/DetailEpics.js

```diff
@@ -82,14 +83,30 @@ export const fetchEpic = (action$, store, call = (fn, ...args) => fn(...args)) =
               ...relationProperties
                 .map(({key, id}) => {
                   const relationProperty = getSchema(state, key);
-                  return getRelationData(relationProperty, id, state);
+
+                  if (relationProperty && id) {
+                    return getRelationData(relationProperty, id, state);
+                  }
+
+                  console.error(`Cannot find schema for ${key}`);
+
+                  return new Promise(resolve => {
+                    resolve('Unknown error.');
```

Comment:
IMHO it should call `reject` instead of `resolve` since it is an error.

Reply:
`reject` would cause throwing an error and stopping fetch epic.
In this case we want to show an error without interrupting render process.